File: src/main/java/net/atos/zac/app/informatieobjecten/model/RESTDocumentVerzendGegevens.java
/*
 * SPDX-FileCopyrightText: 2022 Atos, 2023-2024 Lifely
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.zac.app.informatieobjecten.model;

import java.time.LocalDate;
import java.util.List;
import java.util.UUID;

public class RESTDocumentVerzendGegevens {
  public UUID zaakUuid;

  public LocalDate verzenddatum;

  public List<UUID> informatieobjecten;

  public String toelichting;
}

File: src/main/kotlin/nl/info/zac/app/informatieobjecten/model/RestDocumentVerzendGegevens.kt
/*
 * SPDX-FileCopyrightText: 2023 Atos, 2026 INFO.nl
 * SPDX-License-Identifier: EUPL-1.2+
 */
package nl.info.zac.app.informatieobjecten.model

import nl.info.zac.util.AllOpen
import nl.info.zac.util.NoArgConstructor
import java.time.LocalDate
import java.util.UUID

@NoArgConstructor
@AllOpen
data class RestDocumentVerzendGegevens(
    var zaakUuid: UUID,

    var verzenddatum: LocalDate,

    var informatieobjecten: List<UUID>,

    var toelichting: String? = null
)
